.gitmodules

```diff
@@ -1,26 +1,3 @@
 [submodule "lib/forge-std"]
```

Comment:
why not in npm too?

Reply:
`forge-std` does have a `package.json` but it doesn't track its dependencies. So we would have to also add `ds-test` and keep it in sync. It would also break `forge-std/Test.sol` imports unless we add a remapping. (since the real path is `forge-std/src/Test.sol` )

Worth it to change or should we keep?